locust/main.py

```diff
@@ -115,6 +116,15 @@ def parse_options():
         help="Port that locust master should bind to. Only used when running with --master. Defaults to 5557. Note that Locust will also use this port + 1, so by default the master node will bind to 5557 and 5558."
     )
 
+    parser.add_option(
+        '--expect-slaves',
```

Comment:
Yes, it would but I think that's the behavior I would expect. I think most people using this feature would bring up the master process specifying `--min-slaves 10`, then bring up 10 slave processes to connect to the master. If I wanted 11 slaves, I would simply indicate `--min-slaves 11`. However if I _thought_ I wanted 10 slaves, but later realized I need 11 slaves to reach my desired throughput, then I could add another 1 slave after the original 10 slaves have already started swarming.